app/main.py

```diff
@@ -1 +1,45 @@
-#  write your code here
+class Animal:
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            food_points = self.appetite
+            self.is_hungry = not self.is_hungry
+            return food_points
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super(Cat, self).__init__(name, appetite=3, is_hungry=is_hungry)
```

Comment:
You don't really need to pass `Cat, self` in `super` explicitly. With last updates, it can be used like this `super` and it will automatically take those params